util/topgen/strong_random.py

```diff
@@ -60,7 +65,7 @@ def isempty(self):
         """Checks if the buffer is empty.
         """
 
-        return (self.buffer.size == 0)
+        return (not self.buffer)
```

Comment:
Nice that this works, I would have expected that `self.buffer` just returns the first element.

Reply:
I've just double checked, it indeed works as expected.